rotkehlchen/tests/api/test_uniswap.py

```diff
@@ -732,3 +742,398 @@ def test_get_uniswap_exotic_history(
+                amount1=deserialize_asset_amount('1.046665027131675546'),
+                usd_price=deserialize_price('837.2737746532695970921908229899852'),
+                lp_amount=deserialize_asset_amount('1.220680531244355402'),
+            ),
+        ],
+        profit_loss0=deserialize_asset_amount('-35.489683548121546956'),
+        profit_loss1=deserialize_asset_amount('0.059966416263997186'),
+        usd_profit_loss=deserialize_price('35.19515540870021242982170811'),
+    ),
+]
+EXPECTED_EVENTS_BALANCES_2 = [
+    # Response index 0
+    UniswapPoolEventsBalance(
+        address=deserialize_ethereum_address(TEST_EVENTS_ADDRESS_1),
+        pool_address=deserialize_ethereum_address("0xC585Cc7b9E77AEa3371764320740C18E9aEC9c55"),
+        token0=EthereumToken('WETH'),
+        token1=UnknownEthereumToken(
+            ethereum_address=deserialize_ethereum_address('0xCF67CEd76E8356366291246A9222169F4dBdBe64'),  # noqa: E501
+            symbol='DICE',
+            name='DICE.FINANCE TOKEN',
+        ),
+        events=[
+            UniswapPoolEvent(
+                tx_hash='0x1e7fd116b316af49f6c52b3ca44f3c5d24c2a6f80a5b5e674b5f94155bd2cec4',
+                log_index=99,
+                address=deserialize_ethereum_address(TEST_EVENTS_ADDRESS_1),
+                timestamp=Timestamp(1598270334),
+                event_type=EventType.MINT,
+                pool_address=deserialize_ethereum_address("0xC585Cc7b9E77AEa3371764320740C18E9aEC9c55"),  # noqa: E501
+                token0=EthereumToken('WETH'),
+                token1=UnknownEthereumToken(
+                    ethereum_address=deserialize_ethereum_address('0xCF67CEd76E8356366291246A9222169F4dBdBe64'),  # noqa: E501
+                    symbol='DICE',
+                    name='DICE.FINANCE TOKEN',
+                ),
+                amount0=deserialize_asset_amount('1.580431277572006656'),
+                amount1=deserialize_asset_amount('3'),
+                usd_price=deserialize_price('1281.249386421513581165086356450817'),
+                lp_amount=deserialize_asset_amount('2.074549918528068811'),
+            ),
+            UniswapPoolEvent(
+                tx_hash='0x140bdba831f9494cf0ead6d57009e1eae45ed629a78ee74ccbf49018afae0ffa',
+                log_index=208,
+                address=deserialize_ethereum_address(TEST_EVENTS_ADDRESS_1),
+                timestamp=Timestamp(1599000975),
+                event_type=EventType.BURN,
+                pool_address=deserialize_ethereum_address("0xC585Cc7b9E77AEa3371764320740C18E9aEC9c55"),  # noqa: E501
+                token0=EthereumToken('WETH'),
+                token1=UnknownEthereumToken(
+                    ethereum_address=deserialize_ethereum_address('0xCF67CEd76E8356366291246A9222169F4dBdBe64'),  # noqa: E501
+                    symbol='DICE',
+                    name='DICE.FINANCE TOKEN',
+                ),
+                amount0=deserialize_asset_amount('0.970300671842796406'),
```

Comment:
In all of these perhaps it's an overkill to use the deserialize function? The alternative
```
AssetAmount(FVal('0.5'))
deserialize_asset_amount('0.5')
```

which as you can see is less verbose. But anyway not insisting here since it's already done. I am just pointing out that the test is probably slower (probably not visibly) with all these extra function calls.

Reply:
I wasn't sure about using the "deserialize_" functions in my tests, let me explain. My first batch of Uniswap tests for balances (both integration and unit) didn't call these functions and had all the type conversions in place. Then I saw your trades tests and decided to use them (ok, you didn't use `deserialize_asset_amount`).
Tbh, I've been thinking we should implement in a test utils all these type-conversion functions and do not use the ones from the deserialize module, as they may raise/log errors and have extra logic unrelated to the tests.

Reply:
Ah, well, but for this particular case (and prices, etc.) yes, what I've done is overkilling